{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 66 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 22 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double()\n",
      "  # ... with 23 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer(),\n",
      "  i_End_position_hg18 = col_integer()\n",
      "  # ... with 10 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 52 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 52 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AA = col_logical(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer()\n",
      "  # ... with 39 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number()\n",
      "  # ... with 17 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 23 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 65 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CGC_Mutation_Type = col_logical(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer(),\n",
      "  i_End_position_hg18 = col_integer()\n",
      "  # ... with 10 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 48 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number(),\n",
      "  i_ESP_GS = col_integer(),\n",
      "  i_ESP_Position = col_integer()\n",
      "  # ... with 16 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Warning message:\n",
      "\"1 parsing failure.\n",
      "row col    expected      actual\n",
      " 72  -- 297 columns 187 columns\n",
      "\"Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 23 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 21 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 57 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 22 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double()\n",
      "  # ... with 18 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 22 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double()\n",
      "  # ... with 58 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer(),\n",
      "  i_End_position_hg18 = col_integer()\n",
      "  # ... with 11 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CGC_Mutation_Type = col_logical(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_GS = col_integer()\n",
      "  # ... with 16 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 22 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 21 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 50 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 49 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_Position = col_integer()\n",
      "  # ... with 15 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 47 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 48 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 50 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer(),\n",
      "  i_End_position_hg18 = col_integer()\n",
      "  # ... with 10 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 64 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_GS = col_number()\n",
      "  # ... with 17 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  CGC_Mutation_Type = col_logical(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 52 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number()\n",
      "  # ... with 19 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer()\n",
      "  # ... with 11 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 53 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_Chromosome_hg18 = col_integer(),\n",
      "  i_ESP_AA_AC = col_number(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double()\n",
      "  # ... with 20 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double()\n",
      "  # ... with 32 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer()\n",
      "  # ... with 53 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer()\n",
      "  # ... with 19 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double()\n",
      "  # ... with 57 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 32 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_Chromosome_hg18 = col_integer(),\n",
      "  i_ESP_AA_AC = col_number(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 53 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 56 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 58 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TAC = col_number()\n",
      "  # ... with 42 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 21 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer(),\n",
      "  i_End_position_hg18 = col_integer()\n",
      "  # ... with 11 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 59 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_GS = col_number()\n",
      "  # ... with 17 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double()\n",
      "  # ... with 20 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_Chromosome_hg18 = col_integer(),\n",
      "  i_ESP_AA_AC = col_number(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double()\n",
      "  # ... with 56 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 48 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number()\n",
      "  # ... with 19 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_Chromosome_hg18 = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_Chromosome = col_integer()\n",
      "  # ... with 19 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Sequencing_Phase = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double()\n",
      "  # ... with 55 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 53 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Score = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer(),\n",
      "  i_End_position_hg18 = col_integer(),\n",
      "  i_Entrez_Gene_Id = col_integer(),\n",
      "  `i_HGNC_Date Modified` = col_date(format = \"\")\n",
      "  # ... with 35 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer()\n",
      "  # ... with 48 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 49 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_Chromosome_hg18 = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double()\n",
      "  # ... with 43 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 37 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number()\n",
      "  # ... with 18 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CGC_Mutation_Type = col_logical(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 47 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_Chromosome_hg18 = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double()\n",
      "  # ... with 21 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AA_AC = col_number(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer()\n",
      "  # ... with 18 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 30 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_Chromosome_hg18 = col_integer(),\n",
      "  i_ESP_AA = col_logical(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer()\n",
      "  # ... with 23 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number()\n",
      "  # ... with 17 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 49 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 52 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AA_AC = col_number(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number()\n",
      "  # ... with 46 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer()\n",
      "  # ... with 20 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 51 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 52 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 46 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 55 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 64 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 26 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 52 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 20 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_Position = col_integer(),\n",
      "  i_ESP_TotalAAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalEAsamplesCovered = col_integer(),\n",
      "  i_ESP_TotalSamplesCovered = col_integer(),\n",
      "  i_End_position_hg18 = col_integer()\n",
      "  # ... with 11 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_number(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_Position = col_integer()\n",
      "  # ... with 15 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer()\n",
      "  # ... with 24 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 57 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  Transcript_Position = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 25 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AA_AC = col_number(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CDP = col_integer(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_Position = col_integer()\n",
      "  # ... with 16 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 31 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 29 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Chromosome = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double()\n",
      "  # ... with 28 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_CGC_Chr = col_integer(),\n",
      "  i_CGC_GeneID = col_integer(),\n",
      "  i_COSMIC_n_overlapping_mutations = col_integer(),\n",
      "  i_ESP_AvgAAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgEAsampleReadDepth = col_double(),\n",
      "  i_ESP_AvgSampleReadDepth = col_double(),\n",
      "  i_ESP_CG = col_double(),\n",
      "  i_ESP_CP = col_double(),\n",
      "  i_ESP_DP = col_integer(),\n",
      "  i_ESP_EA_AC = col_number(),\n",
      "  i_ESP_Position = col_integer()\n",
      "  # ... with 15 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n",
      "Parsed with column specification:\n",
      "cols(\n",
      "  .default = col_character(),\n",
      "  Entrez_Gene_Id = col_integer(),\n",
      "  NCBI_Build = col_integer(),\n",
      "  Start_position = col_integer(),\n",
      "  End_position = col_integer(),\n",
      "  Transcript_Exon = col_integer(),\n",
      "  UniProt_AApos = col_integer(),\n",
      "  COSMIC_total_alterations_in_gene = col_integer(),\n",
      "  gc_content = col_double(),\n",
      "  CCLE_ONCOMAP_total_mutations_in_gene = col_integer(),\n",
      "  i_1000gp3_AC = col_integer(),\n",
      "  i_1000gp3_AF = col_double(),\n",
      "  i_1000gp3_AFR_AF = col_double(),\n",
      "  i_1000gp3_AMR_AF = col_double(),\n",
      "  i_1000gp3_AN = col_integer(),\n",
      "  i_1000gp3_DP = col_integer(),\n",
      "  i_1000gp3_EAS_AF = col_double(),\n",
      "  i_1000gp3_EUR_AF = col_double(),\n",
      "  i_1000gp3_NS = col_integer(),\n",
      "  i_1000gp3_SAS_AF = col_double(),\n",
      "  i_CGC_Chr = col_integer()\n",
      "  # ... with 27 more columns\n",
      ")\n",
      "See spec(...) for full column specifications.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Hugo_Symbol</th><th scope=col>Entrez_Gene_Id</th><th scope=col>Center</th><th scope=col>NCBI_Build</th><th scope=col>Chromosome</th><th scope=col>Start_position</th><th scope=col>End_position</th><th scope=col>Strand</th><th scope=col>Variant_Classification</th><th scope=col>Variant_Type</th><th scope=col>...</th><th scope=col>i_gc_content_full</th><th scope=col>i_gencode_transcript_name</th><th scope=col>i_gencode_transcript_status</th><th scope=col>i_gencode_transcript_tags</th><th scope=col>i_gencode_transcript_type</th><th scope=col>i_gene_type</th><th scope=col>i_havana_transcript</th><th scope=col>i_refseq_mrna_id</th><th scope=col>i_secondary_variant_classification</th><th scope=col>i_type</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>BAI2                               </td><td>   576                             </td><td>broad.mit.edu                      </td><td>37                                 </td><td>1                                  </td><td> 32202221                          </td><td> 32202221                          </td><td>+                                  </td><td>Missense_Mutation                  </td><td>SNP                                </td><td>...                                </td><td>0.642                              </td><td>BAI2-015                           </td><td>KNOWN                              </td><td>basic|CCDS                         </td><td>protein_coding                     </td><td>protein_coding                     </td><td>OTTHUMT00000381838.1               </td><td>NM_001703                          </td><td>NA                                 </td><td>Missense_Mutation                  </td></tr>\n",
       "\t<tr><td>LRRC41                             </td><td> 10489                             </td><td>broad.mit.edu                      </td><td>37                                 </td><td>1                                  </td><td> 46751997                          </td><td> 46751997                          </td><td>+                                  </td><td>Missense_Mutation                  </td><td>SNP                                </td><td>...                                </td><td>0.592                              </td><td>LRRC41-001                         </td><td>KNOWN                              </td><td>basic|appris_principal|CCDS        </td><td>protein_coding                     </td><td>protein_coding                     </td><td>OTTHUMT00000021438.1               </td><td>NM_006369                          </td><td>NA                                 </td><td>Missense_Mutation                  </td></tr>\n",
       "\t<tr><td>ERICH3                             </td><td>127254                             </td><td>broad.mit.edu                      </td><td>37                                 </td><td>1                                  </td><td> 75055494                          </td><td> 75055494                          </td><td>+                                  </td><td>Missense_Mutation                  </td><td>SNP                                </td><td>...                                </td><td>0.408                              </td><td>C1orf173-001                       </td><td>KNOWN                              </td><td>basic|appris_principal|CCDS        </td><td>protein_coding                     </td><td>protein_coding                     </td><td>OTTHUMT00000026516.1               </td><td>NA                                 </td><td>NA                                 </td><td>Missense_Mutation                  </td></tr>\n",
       "\t<tr><td>SSX2IP                             </td><td>117178                             </td><td>broad.mit.edu                      </td><td>37                                 </td><td>1                                  </td><td> 85128152                          </td><td> 85128152                          </td><td>+                                  </td><td>Silent                             </td><td>SNP                                </td><td>...                                </td><td>0.363                              </td><td>SSX2IP-001                         </td><td>KNOWN                              </td><td>basic|appris_principal|CCDS        </td><td>protein_coding                     </td><td>protein_coding                     </td><td>OTTHUMT00000027469.1               </td><td>NM_014021                          </td><td>NA                                 </td><td>Silent                             </td></tr>\n",
       "\t<tr><td>NTNG1                              </td><td> 22854                             </td><td>broad.mit.edu                      </td><td>37                                 </td><td>1                                  </td><td>107866918                          </td><td>107866918                          </td><td>+                                  </td><td>Missense_Mutation                  </td><td>SNP                                </td><td>...                                </td><td>0.478                              </td><td>NTNG1-006                          </td><td>KNOWN                              </td><td>basic|appris_candidate_longest|CCDS</td><td>protein_coding                     </td><td>protein_coding                     </td><td>OTTHUMT00000030340.1               </td><td>NM_014917                          </td><td>NA                                 </td><td>Missense_Mutation                  </td></tr>\n",
       "\t<tr><td>PGLYRP3                            </td><td>114771                             </td><td>broad.mit.edu                      </td><td>37                                 </td><td>1                                  </td><td>153274927                          </td><td>153274927                          </td><td>+                                  </td><td>Missense_Mutation                  </td><td>SNP                                </td><td>...                                </td><td>0.458                              </td><td>PGLYRP3-001                        </td><td>KNOWN                              </td><td>basic|appris_principal|CCDS        </td><td>protein_coding                     </td><td>protein_coding                     </td><td>OTTHUMT00000039488.1               </td><td>NM_052891                          </td><td>NA                                 </td><td>Missense_Mutation                  </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll}\n",
       " Hugo\\_Symbol & Entrez\\_Gene\\_Id & Center & NCBI\\_Build & Chromosome & Start\\_position & End\\_position & Strand & Variant\\_Classification & Variant\\_Type & ... & i\\_gc\\_content\\_full & i\\_gencode\\_transcript\\_name & i\\_gencode\\_transcript\\_status & i\\_gencode\\_transcript\\_tags & i\\_gencode\\_transcript\\_type & i\\_gene\\_type & i\\_havana\\_transcript & i\\_refseq\\_mrna\\_id & i\\_secondary\\_variant\\_classification & i\\_type\\\\\n",
       "\\hline\n",
       "\t BAI2                                          &    576                                        & broad.mit.edu                                 & 37                                            & 1                                             &  32202221                                     &  32202221                                     & +                                             & Missense\\_Mutation                           & SNP                                           & ...                                           & 0.642                                         & BAI2-015                                      & KNOWN                                         & basic\\textbar{}CCDS                          & protein\\_coding                              & protein\\_coding                              & OTTHUMT00000381838.1                          & NM\\_001703                                   & NA                                            & Missense\\_Mutation                          \\\\\n",
       "\t LRRC41                                                    &  10489                                                    & broad.mit.edu                                             & 37                                                        & 1                                                         &  46751997                                                 &  46751997                                                 & +                                                         & Missense\\_Mutation                                       & SNP                                                       & ...                                                       & 0.592                                                     & LRRC41-001                                                & KNOWN                                                     & basic\\textbar{}appris\\_principal\\textbar{}CCDS         & protein\\_coding                                          & protein\\_coding                                          & OTTHUMT00000021438.1                                      & NM\\_006369                                               & NA                                                        & Missense\\_Mutation                                      \\\\\n",
       "\t ERICH3                                                    & 127254                                                    & broad.mit.edu                                             & 37                                                        & 1                                                         &  75055494                                                 &  75055494                                                 & +                                                         & Missense\\_Mutation                                       & SNP                                                       & ...                                                       & 0.408                                                     & C1orf173-001                                              & KNOWN                                                     & basic\\textbar{}appris\\_principal\\textbar{}CCDS         & protein\\_coding                                          & protein\\_coding                                          & OTTHUMT00000026516.1                                      & NA                                                        & NA                                                        & Missense\\_Mutation                                      \\\\\n",
       "\t SSX2IP                                                    & 117178                                                    & broad.mit.edu                                             & 37                                                        & 1                                                         &  85128152                                                 &  85128152                                                 & +                                                         & Silent                                                    & SNP                                                       & ...                                                       & 0.363                                                     & SSX2IP-001                                                & KNOWN                                                     & basic\\textbar{}appris\\_principal\\textbar{}CCDS         & protein\\_coding                                          & protein\\_coding                                          & OTTHUMT00000027469.1                                      & NM\\_014021                                               & NA                                                        & Silent                                                   \\\\\n",
       "\t NTNG1                                                       &  22854                                                      & broad.mit.edu                                               & 37                                                          & 1                                                           & 107866918                                                   & 107866918                                                   & +                                                           & Missense\\_Mutation                                         & SNP                                                         & ...                                                         & 0.478                                                       & NTNG1-006                                                   & KNOWN                                                       & basic\\textbar{}appris\\_candidate\\_longest\\textbar{}CCDS & protein\\_coding                                            & protein\\_coding                                            & OTTHUMT00000030340.1                                        & NM\\_014917                                                 & NA                                                          & Missense\\_Mutation                                        \\\\\n",
       "\t PGLYRP3                                                   & 114771                                                    & broad.mit.edu                                             & 37                                                        & 1                                                         & 153274927                                                 & 153274927                                                 & +                                                         & Missense\\_Mutation                                       & SNP                                                       & ...                                                       & 0.458                                                     & PGLYRP3-001                                               & KNOWN                                                     & basic\\textbar{}appris\\_principal\\textbar{}CCDS         & protein\\_coding                                          & protein\\_coding                                          & OTTHUMT00000039488.1                                      & NM\\_052891                                               & NA                                                        & Missense\\_Mutation                                      \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Hugo_Symbol | Entrez_Gene_Id | Center | NCBI_Build | Chromosome | Start_position | End_position | Strand | Variant_Classification | Variant_Type | ... | i_gc_content_full | i_gencode_transcript_name | i_gencode_transcript_status | i_gencode_transcript_tags | i_gencode_transcript_type | i_gene_type | i_havana_transcript | i_refseq_mrna_id | i_secondary_variant_classification | i_type | \n",
       "|---|---|---|---|---|---|\n",
       "| BAI2                                |    576                              | broad.mit.edu                       | 37                                  | 1                                   |  32202221                           |  32202221                           | +                                   | Missense_Mutation                   | SNP                                 | ...                                 | 0.642                               | BAI2-015                            | KNOWN                               | basic|CCDS                          | protein_coding                      | protein_coding                      | OTTHUMT00000381838.1                | NM_001703                           | NA                                  | Missense_Mutation                   | \n",
       "| LRRC41                              |  10489                              | broad.mit.edu                       | 37                                  | 1                                   |  46751997                           |  46751997                           | +                                   | Missense_Mutation                   | SNP                                 | ...                                 | 0.592                               | LRRC41-001                          | KNOWN                               | basic|appris_principal|CCDS         | protein_coding                      | protein_coding                      | OTTHUMT00000021438.1                | NM_006369                           | NA                                  | Missense_Mutation                   | \n",
       "| ERICH3                              | 127254                              | broad.mit.edu                       | 37                                  | 1                                   |  75055494                           |  75055494                           | +                                   | Missense_Mutation                   | SNP                                 | ...                                 | 0.408                               | C1orf173-001                        | KNOWN                               | basic|appris_principal|CCDS         | protein_coding                      | protein_coding                      | OTTHUMT00000026516.1                | NA                                  | NA                                  | Missense_Mutation                   | \n",
       "| SSX2IP                              | 117178                              | broad.mit.edu                       | 37                                  | 1                                   |  85128152                           |  85128152                           | +                                   | Silent                              | SNP                                 | ...                                 | 0.363                               | SSX2IP-001                          | KNOWN                               | basic|appris_principal|CCDS         | protein_coding                      | protein_coding                      | OTTHUMT00000027469.1                | NM_014021                           | NA                                  | Silent                              | \n",
       "| NTNG1                               |  22854                              | broad.mit.edu                       | 37                                  | 1                                   | 107866918                           | 107866918                           | +                                   | Missense_Mutation                   | SNP                                 | ...                                 | 0.478                               | NTNG1-006                           | KNOWN                               | basic|appris_candidate_longest|CCDS | protein_coding                      | protein_coding                      | OTTHUMT00000030340.1                | NM_014917                           | NA                                  | Missense_Mutation                   | \n",
       "| PGLYRP3                             | 114771                              | broad.mit.edu                       | 37                                  | 1                                   | 153274927                           | 153274927                           | +                                   | Missense_Mutation                   | SNP                                 | ...                                 | 0.458                               | PGLYRP3-001                         | KNOWN                               | basic|appris_principal|CCDS         | protein_coding                      | protein_coding                      | OTTHUMT00000039488.1                | NM_052891                           | NA                                  | Missense_Mutation                   | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Hugo_Symbol Entrez_Gene_Id Center        NCBI_Build Chromosome Start_position\n",
       "1 BAI2           576         broad.mit.edu 37         1           32202221     \n",
       "2 LRRC41       10489         broad.mit.edu 37         1           46751997     \n",
       "3 ERICH3      127254         broad.mit.edu 37         1           75055494     \n",
       "4 SSX2IP      117178         broad.mit.edu 37         1           85128152     \n",
       "5 NTNG1        22854         broad.mit.edu 37         1          107866918     \n",
       "6 PGLYRP3     114771         broad.mit.edu 37         1          153274927     \n",
       "  End_position Strand Variant_Classification Variant_Type ... i_gc_content_full\n",
       "1  32202221    +      Missense_Mutation      SNP          ... 0.642            \n",
       "2  46751997    +      Missense_Mutation      SNP          ... 0.592            \n",
       "3  75055494    +      Missense_Mutation      SNP          ... 0.408            \n",
       "4  85128152    +      Silent                 SNP          ... 0.363            \n",
       "5 107866918    +      Missense_Mutation      SNP          ... 0.478            \n",
       "6 153274927    +      Missense_Mutation      SNP          ... 0.458            \n",
       "  i_gencode_transcript_name i_gencode_transcript_status\n",
       "1 BAI2-015                  KNOWN                      \n",
       "2 LRRC41-001                KNOWN                      \n",
       "3 C1orf173-001              KNOWN                      \n",
       "4 SSX2IP-001                KNOWN                      \n",
       "5 NTNG1-006                 KNOWN                      \n",
       "6 PGLYRP3-001               KNOWN                      \n",
       "  i_gencode_transcript_tags           i_gencode_transcript_type i_gene_type   \n",
       "1 basic|CCDS                          protein_coding            protein_coding\n",
       "2 basic|appris_principal|CCDS         protein_coding            protein_coding\n",
       "3 basic|appris_principal|CCDS         protein_coding            protein_coding\n",
       "4 basic|appris_principal|CCDS         protein_coding            protein_coding\n",
       "5 basic|appris_candidate_longest|CCDS protein_coding            protein_coding\n",
       "6 basic|appris_principal|CCDS         protein_coding            protein_coding\n",
       "  i_havana_transcript  i_refseq_mrna_id i_secondary_variant_classification\n",
       "1 OTTHUMT00000381838.1 NM_001703        NA                                \n",
       "2 OTTHUMT00000021438.1 NM_006369        NA                                \n",
       "3 OTTHUMT00000026516.1 NA               NA                                \n",
       "4 OTTHUMT00000027469.1 NM_014021        NA                                \n",
       "5 OTTHUMT00000030340.1 NM_014917        NA                                \n",
       "6 OTTHUMT00000039488.1 NM_052891        NA                                \n",
       "  i_type           \n",
       "1 Missense_Mutation\n",
       "2 Missense_Mutation\n",
       "3 Missense_Mutation\n",
       "4 Silent           \n",
       "5 Missense_Mutation\n",
       "6 Missense_Mutation"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# we will need these packages that you download from Bioconductor('maftools') and CRAN ('readr', 'NMF')\n",
    "library(readr)\n",
    "library(NMF)\n",
    "library(maftools)\n",
    "\n",
    "# change this path according to where you created the folder\n",
    "#dataPath <- \"C:/-=TRANSMED=-/course materials/teaching/BIOINF17/ex1/bioinf/\" \n",
    "fileList <- list.files( paste0(\"gdac.broadinstitute.org_OV.Mutation_Packager_Oncotated_Calls.Level_3.2016012800.0.0/\"))\n",
    "\n",
    "# this is to remove the manifest file from the list\n",
    "fileList <- fileList[ fileList != \"MANIFEST.txt\"] \n",
    "\n",
    "# let's create an empty list and then populate it with the files\n",
    "mafList <- list()\n",
    "for (i in seq_along(fileList)) \n",
    "  mafList[[i]] <- read_tsv( paste0( \n",
    "                \"gdac.broadinstitute.org_OV.Mutation_Packager_Oncotated_Calls.Level_3.2016012800.0.0/\",\n",
    "                fileList[i]) ,comment = \"#\", progress = F)\n",
    "\n",
    "#now let's aggregate the list into one data frame\n",
    "maf.df <- as.data.frame(do.call(rbind, mafList))\n",
    "head(maf.df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  [1] Hugo_Symbol                                  \n",
      "  [2] Entrez_Gene_Id                               \n",
      "  [3] Center                                       \n",
      "  [4] NCBI_Build                                   \n",
      "  [5] Chromosome                                   \n",
      "  [6] Start_position                               \n",
      "  [7] End_position                                 \n",
      "  [8] Strand                                       \n",
      "  [9] Variant_Classification                       \n",
      " [10] Variant_Type                                 \n",
      " [11] Reference_Allele                             \n",
      " [12] Tumor_Seq_Allele1                            \n",
      " [13] Tumor_Seq_Allele2                            \n",
      " [14] dbSNP_RS                                     \n",
      " [15] dbSNP_Val_Status                             \n",
      " [16] Tumor_Sample_Barcode                         \n",
      " [17] Matched_Norm_Sample_Barcode                  \n",
      " [18] Match_Norm_Seq_Allele1                       \n",
      " [19] Match_Norm_Seq_Allele2                       \n",
      " [20] Tumor_Validation_Allele1                     \n",
      " [21] Tumor_Validation_Allele2                     \n",
      " [22] Match_Norm_Validation_Allele1                \n",
      " [23] Match_Norm_Validation_Allele2                \n",
      " [24] Verification_Status                          \n",
      " [25] Validation_Status                            \n",
      " [26] Mutation_Status                              \n",
      " [27] Sequencing_Phase                             \n",
      " [28] Sequence_Source                              \n",
      " [29] Validation_Method                            \n",
      " [30] Score                                        \n",
      " [31] BAM_file                                     \n",
      " [32] Sequencer                                    \n",
      " [33] Tumor_Sample_UUID                            \n",
      " [34] Matched_Norm_Sample_UUID                     \n",
      " [35] Genome_Change                                \n",
      " [36] Annotation_Transcript                        \n",
      " [37] Transcript_Strand                            \n",
      " [38] Transcript_Exon                              \n",
      " [39] Transcript_Position                          \n",
      " [40] cDNA_Change                                  \n",
      " [41] Codon_Change                                 \n",
      " [42] Protein_Change                               \n",
      " [43] Other_Transcripts                            \n",
      " [44] Refseq_mRNA_Id                               \n",
      " [45] Refseq_prot_Id                               \n",
      " [46] SwissProt_acc_Id                             \n",
      " [47] SwissProt_entry_Id                           \n",
      " [48] Description                                  \n",
      " [49] UniProt_AApos                                \n",
      " [50] UniProt_Region                               \n",
      " [51] UniProt_Site                                 \n",
      " [52] UniProt_Natural_Variations                   \n",
      " [53] UniProt_Experimental_Info                    \n",
      " [54] GO_Biological_Process                        \n",
      " [55] GO_Cellular_Component                        \n",
      " [56] GO_Molecular_Function                        \n",
      " [57] COSMIC_overlapping_mutations                 \n",
      " [58] COSMIC_fusion_genes                          \n",
      " [59] COSMIC_tissue_types_affected                 \n",
      " [60] COSMIC_total_alterations_in_gene             \n",
      " [61] Tumorscape_Amplification_Peaks               \n",
      " [62] Tumorscape_Deletion_Peaks                    \n",
      " [63] TCGAscape_Amplification_Peaks                \n",
      " [64] TCGAscape_Deletion_Peaks                     \n",
      " [65] DrugBank                                     \n",
      " [66] ref_context                                  \n",
      " [67] gc_content                                   \n",
      " [68] CCLE_ONCOMAP_overlapping_mutations           \n",
      " [69] CCLE_ONCOMAP_total_mutations_in_gene         \n",
      " [70] CGC_Mutation_Type                            \n",
      " [71] CGC_Translocation_Partner                    \n",
      " [72] CGC_Tumor_Types_Somatic                      \n",
      " [73] CGC_Tumor_Types_Germline                     \n",
      " [74] CGC_Other_Diseases                           \n",
      " [75] DNARepairGenes_Role                          \n",
      " [76] FamilialCancerDatabase_Syndromes             \n",
      " [77] MUTSIG_Published_Results                     \n",
      " [78] OREGANNO_ID                                  \n",
      " [79] OREGANNO_Values                              \n",
      " [80] i_1000gp3_AA                                 \n",
      " [81] i_1000gp3_AC                                 \n",
      " [82] i_1000gp3_AF                                 \n",
      " [83] i_1000gp3_AFR_AF                             \n",
      " [84] i_1000gp3_AMR_AF                             \n",
      " [85] i_1000gp3_AN                                 \n",
      " [86] i_1000gp3_CIEND                              \n",
      " [87] i_1000gp3_CIPOS                              \n",
      " [88] i_1000gp3_CS                                 \n",
      " [89] i_1000gp3_DP                                 \n",
      " [90] i_1000gp3_EAS_AF                             \n",
      " [91] i_1000gp3_END                                \n",
      " [92] i_1000gp3_EUR_AF                             \n",
      " [93] i_1000gp3_IMPRECISE                          \n",
      " [94] i_1000gp3_MC                                 \n",
      " [95] i_1000gp3_MEINFO                             \n",
      " [96] i_1000gp3_MEND                               \n",
      " [97] i_1000gp3_MLEN                               \n",
      " [98] i_1000gp3_MSTART                             \n",
      " [99] i_1000gp3_NS                                 \n",
      "[100] i_1000gp3_SAS_AF                             \n",
      "[101] i_1000gp3_SVLEN                              \n",
      "[102] i_1000gp3_SVTYPE                             \n",
      "[103] i_1000gp3_TSD                                \n",
      "[104] i_ACHILLES_Lineage_Results_Top_Genes         \n",
      "[105] i_CGC_Cancer Germline Mut                    \n",
      "[106] i_CGC_Cancer Molecular Genetics              \n",
      "[107] i_CGC_Cancer Somatic Mut                     \n",
      "[108] i_CGC_Cancer Syndrome                        \n",
      "[109] i_CGC_Chr                                    \n",
      "[110] i_CGC_Chr Band                               \n",
      "[111] i_CGC_GeneID                                 \n",
      "[112] i_CGC_Name                                   \n",
      "[113] i_CGC_Other Germline Mut                     \n",
      "[114] i_CGC_Tissue Type                            \n",
      "[115] i_COSMIC_n_overlapping_mutations             \n",
      "[116] i_COSMIC_overlapping_mutation_descriptions   \n",
      "[117] i_COSMIC_overlapping_primary_sites           \n",
      "[118] i_Chromosome_hg18                            \n",
      "[119] i_ClinVar_ASSEMBLY                           \n",
      "[120] i_ClinVar_HGMD_ID                            \n",
      "[121] i_ClinVar_SYM                                \n",
      "[122] i_ClinVar_TYPE                               \n",
      "[123] i_ClinVar_rs                                 \n",
      "[124] i_ESP_AA                                     \n",
      "[125] i_ESP_AAC                                    \n",
      "[126] i_ESP_AA_AC                                  \n",
      "[127] i_ESP_AA_AGE                                 \n",
      "[128] i_ESP_AA_GTC                                 \n",
      "[129] i_ESP_AvgAAsampleReadDepth                   \n",
      "[130] i_ESP_AvgEAsampleReadDepth                   \n",
      "[131] i_ESP_AvgSampleReadDepth                     \n",
      "[132] i_ESP_CA                                     \n",
      "[133] i_ESP_CDP                                    \n",
      "[134] i_ESP_CG                                     \n",
      "[135] i_ESP_CP                                     \n",
      "[136] i_ESP_Chromosome                             \n",
      "[137] i_ESP_DBSNP                                  \n",
      "[138] i_ESP_DP                                     \n",
      "[139] i_ESP_EA_AC                                  \n",
      "[140] i_ESP_EA_AGE                                 \n",
      "[141] i_ESP_EA_GTC                                 \n",
      "[142] i_ESP_EXOME_CHIP                             \n",
      "[143] i_ESP_FG                                     \n",
      "[144] i_ESP_GL                                     \n",
      "[145] i_ESP_GM                                     \n",
      "[146] i_ESP_GS                                     \n",
      "[147] i_ESP_GTC                                    \n",
      "[148] i_ESP_GTS                                    \n",
      "[149] i_ESP_GWAS_PUBMED                            \n",
      "[150] i_ESP_MAF                                    \n",
      "[151] i_ESP_PH                                     \n",
      "[152] i_ESP_PP                                     \n",
      "[153] i_ESP_Position                               \n",
      "[154] i_ESP_TAC                                    \n",
      "[155] i_ESP_TotalAAsamplesCovered                  \n",
      "[156] i_ESP_TotalEAsamplesCovered                  \n",
      "[157] i_ESP_TotalSamplesCovered                    \n",
      "[158] i_End_position_hg18                          \n",
      "[159] i_Ensembl_so_accession                       \n",
      "[160] i_Ensembl_so_term                            \n",
      "[161] i_Entrez_Gene_Id                             \n",
      "[162] i_Familial_Cancer_Genes_Reference            \n",
      "[163] i_Familial_Cancer_Genes_Synonym              \n",
      "[164] i_HGNC_Accession Numbers                     \n",
      "[165] i_HGNC_CCDS IDs                              \n",
      "[166] i_HGNC_Chromosome                            \n",
      "[167] i_HGNC_Date Modified                         \n",
      "[168] i_HGNC_Date Name Changed                     \n",
      "[169] i_HGNC_Date Symbol Changed                   \n",
      "[170] i_HGNC_Ensembl Gene ID                       \n",
      "[171] i_HGNC_Ensembl ID(supplied by Ensembl)       \n",
      "[172] i_HGNC_Enzyme IDs                            \n",
      "[173] i_HGNC_Gene family description               \n",
      "[174] i_HGNC_HGNC ID                               \n",
      "[175] i_HGNC_Locus Group                           \n",
      "[176] i_HGNC_Locus Type                            \n",
      "[177] i_HGNC_Name Synonyms                         \n",
      "[178] i_HGNC_OMIM ID(supplied by NCBI)             \n",
      "[179] i_HGNC_Previous Names                        \n",
      "[180] i_HGNC_Previous Symbols                      \n",
      "[181] i_HGNC_Primary IDs                           \n",
      "[182] i_HGNC_Pubmed IDs                            \n",
      "[183] i_HGNC_Record Type                           \n",
      "[184] i_HGNC_RefSeq(supplied by NCBI)              \n",
      "[185] i_HGNC_Secondary IDs                         \n",
      "[186] i_HGNC_Status                                \n",
      "[187] i_HGNC_Synonyms                              \n",
      "[188] i_HGNC_UCSC ID(supplied by UCSC)             \n",
      "[189] i_HGNC_UniProt ID(supplied by UniProt)       \n",
      "[190] i_HGNC_VEGA IDs                              \n",
      "[191] i_HGVS_coding_DNA_change                     \n",
      "[192] i_HGVS_genomic_change                        \n",
      "[193] i_HGVS_protein_change                        \n",
      "[194] i_ORegAnno_bin                               \n",
      "[195] i_Start_position_hg18                        \n",
      "[196] i_UniProt_alt_uniprot_accessions             \n",
      "[197] i_Variant_Classification                     \n",
      "[198] i_Variant_Type                               \n",
      "[199] i_annotation_transcript                      \n",
      "[200] i_build                                      \n",
      "[201] i_ccds_id                                    \n",
      "[202] i_classification                             \n",
      "[203] i_context                                    \n",
      "[204] i_dataset                                    \n",
      "[205] i_dbNSFP_1000Gp1_AC                          \n",
      "[206] i_dbNSFP_1000Gp1_AF                          \n",
      "[207] i_dbNSFP_1000Gp1_AFR_AC                      \n",
      "[208] i_dbNSFP_1000Gp1_AFR_AF                      \n",
      "[209] i_dbNSFP_1000Gp1_AMR_AC                      \n",
      "[210] i_dbNSFP_1000Gp1_AMR_AF                      \n",
      "[211] i_dbNSFP_1000Gp1_ASN_AC                      \n",
      "[212] i_dbNSFP_1000Gp1_ASN_AF                      \n",
      "[213] i_dbNSFP_1000Gp1_EUR_AC                      \n",
      "[214] i_dbNSFP_1000Gp1_EUR_AF                      \n",
      "[215] i_dbNSFP_Ancestral_allele                    \n",
      "[216] i_dbNSFP_CADD_phred                          \n",
      "[217] i_dbNSFP_CADD_raw                            \n",
      "[218] i_dbNSFP_CADD_raw_rankscore                  \n",
      "[219] i_dbNSFP_ESP6500_AA_AF                       \n",
      "[220] i_dbNSFP_ESP6500_EA_AF                       \n",
      "[221] i_dbNSFP_Ensembl_geneid                      \n",
      "[222] i_dbNSFP_Ensembl_transcriptid                \n",
      "[223] i_dbNSFP_FATHMM_pred                         \n",
      "[224] i_dbNSFP_FATHMM_rankscore                    \n",
      "[225] i_dbNSFP_FATHMM_score                        \n",
      "[226] i_dbNSFP_GERP++_NR                           \n",
      "[227] i_dbNSFP_GERP++_RS                           \n",
      "[228] i_dbNSFP_GERP++_RS_rankscore                 \n",
      "[229] i_dbNSFP_Interpro_domain                     \n",
      "[230] i_dbNSFP_LRT_Omega                           \n",
      "[231] i_dbNSFP_LRT_converted_rankscore             \n",
      "[232] i_dbNSFP_LRT_pred                            \n",
      "[233] i_dbNSFP_LRT_score                           \n",
      "[234] i_dbNSFP_LR_pred                             \n",
      "[235] i_dbNSFP_LR_rankscore                        \n",
      "[236] i_dbNSFP_LR_score                            \n",
      "[237] i_dbNSFP_MutationAssessor_pred               \n",
      "[238] i_dbNSFP_MutationAssessor_rankscore          \n",
      "[239] i_dbNSFP_MutationAssessor_score              \n",
      "[240] i_dbNSFP_MutationTaster_converted_rankscore  \n",
      "[241] i_dbNSFP_MutationTaster_pred                 \n",
      "[242] i_dbNSFP_MutationTaster_score                \n",
      "[243] i_dbNSFP_Polyphen2_HDIV_pred                 \n",
      "[244] i_dbNSFP_Polyphen2_HDIV_rankscore            \n",
      "[245] i_dbNSFP_Polyphen2_HDIV_score                \n",
      "[246] i_dbNSFP_Polyphen2_HVAR_pred                 \n",
      "[247] i_dbNSFP_Polyphen2_HVAR_rankscore            \n",
      "[248] i_dbNSFP_Polyphen2_HVAR_score                \n",
      "[249] i_dbNSFP_RadialSVM_pred                      \n",
      "[250] i_dbNSFP_RadialSVM_rankscore                 \n",
      "[251] i_dbNSFP_RadialSVM_score                     \n",
      "[252] i_dbNSFP_Reliability_index                   \n",
      "[253] i_dbNSFP_SIFT_converted_rankscore            \n",
      "[254] i_dbNSFP_SIFT_pred                           \n",
      "[255] i_dbNSFP_SIFT_score                          \n",
      "[256] i_dbNSFP_SLR_test_statistic                  \n",
      "[257] i_dbNSFP_SiPhy_29way_logOdds                 \n",
      "[258] i_dbNSFP_SiPhy_29way_logOdds_rankscore       \n",
      "[259] i_dbNSFP_SiPhy_29way_pi                      \n",
      "[260] i_dbNSFP_UniSNP_ids                          \n",
      "[261] i_dbNSFP_Uniprot_aapos                       \n",
      "[262] i_dbNSFP_Uniprot_acc                         \n",
      "[263] i_dbNSFP_Uniprot_id                          \n",
      "[264] i_dbNSFP_aaalt                               \n",
      "[265] i_dbNSFP_aapos                               \n",
      "[266] i_dbNSFP_aapos_FATHMM                        \n",
      "[267] i_dbNSFP_aapos_SIFT                          \n",
      "[268] i_dbNSFP_aaref                               \n",
      "[269] i_dbNSFP_cds_strand                          \n",
      "[270] i_dbNSFP_codonpos                            \n",
      "[271] i_dbNSFP_fold-degenerate                     \n",
      "[272] i_dbNSFP_genename                            \n",
      "[273] i_dbNSFP_hg18_pos(1-coor)                    \n",
      "[274] i_dbNSFP_phastCons100way_vertebrate          \n",
      "[275] i_dbNSFP_phastCons100way_vertebrate_rankscore\n",
      "[276] i_dbNSFP_phastCons46way_placental            \n",
      "[277] i_dbNSFP_phastCons46way_placental_rankscore  \n",
      "[278] i_dbNSFP_phastCons46way_primate              \n",
      "[279] i_dbNSFP_phastCons46way_primate_rankscore    \n",
      "[280] i_dbNSFP_phyloP100way_vertebrate             \n",
      "[281] i_dbNSFP_phyloP100way_vertebrate_rankscore   \n",
      "[282] i_dbNSFP_phyloP46way_placental               \n",
      "[283] i_dbNSFP_phyloP46way_placental_rankscore     \n",
      "[284] i_dbNSFP_phyloP46way_primate                 \n",
      "[285] i_dbNSFP_phyloP46way_primate_rankscore       \n",
      "[286] i_dbNSFP_refcodon                            \n",
      "[287] i_entrez_gene_id                             \n",
      "[288] i_gc_content_full                            \n",
      "[289] i_gencode_transcript_name                    \n",
      "[290] i_gencode_transcript_status                  \n",
      "[291] i_gencode_transcript_tags                    \n",
      "[292] i_gencode_transcript_type                    \n",
      "[293] i_gene_type                                  \n",
      "[294] i_havana_transcript                          \n",
      "[295] i_refseq_mrna_id                             \n",
      "[296] i_secondary_variant_classification           \n",
      "[297] i_type                                       \n",
      "<0 rows> (or 0-length row.names)\n"
     ]
    }
   ],
   "source": [
    "print(maf.df[0,])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# You can check here and appreciate how many possible annotations one can get\n",
    "# We will choose specific columns only for ease of use\n",
    "\n",
    "selectedColumns <- c( \"Hugo_Symbol\", \"Center\", \"NCBI_Build\", \"Chromosome\",\n",
    "                      \"Start_position\", \"End_position\", \"Variant_Classification\",\n",
    "                      \"Variant_Type\", \"Tumor_Sample_Barcode\", \"Mutation_Status\", \n",
    "                      \"Reference_Allele\", \"Tumor_Seq_Allele1\", \"Tumor_Seq_Allele2\",\n",
    "                      \"Protein_Change\", \"i_dbNSFP_CADD_phred\")\n",
    "\n",
    "maf.df <- maf.df[, selectedColumns]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "'data.frame':\t20160 obs. of  15 variables:\n",
      " $ Hugo_Symbol           : chr  \"BAI2\" \"LRRC41\" \"ERICH3\" \"SSX2IP\" ...\n",
      " $ Center                : chr  \"broad.mit.edu\" \"broad.mit.edu\" \"broad.mit.edu\" \"broad.mit.edu\" ...\n",
      " $ NCBI_Build            : int  37 37 37 37 37 37 37 37 37 37 ...\n",
      " $ Chromosome            : chr  \"1\" \"1\" \"1\" \"1\" ...\n",
      " $ Start_position        : int  32202221 46751997 75055494 85128152 107866918 153274927 157516843 224321795 248343719 28023684 ...\n",
      " $ End_position          : int  32202221 46751997 75055494 85128152 107866918 153274927 157516843 224321795 248343719 28023684 ...\n",
      " $ Variant_Classification: chr  \"Missense_Mutation\" \"Missense_Mutation\" \"Missense_Mutation\" \"Silent\" ...\n",
      " $ Variant_Type          : chr  \"SNP\" \"SNP\" \"SNP\" \"SNP\" ...\n",
      " $ Tumor_Sample_Barcode  : chr  \"TCGA-04-1331-01\" \"TCGA-04-1331-01\" \"TCGA-04-1331-01\" \"TCGA-04-1331-01\" ...\n",
      " $ Mutation_Status       : chr  \"Unknown\" \"Somatic\" \"Unknown\" \"Somatic\" ...\n",
      " $ Reference_Allele      : chr  \"C\" \"C\" \"T\" \"A\" ...\n",
      " $ Tumor_Seq_Allele1     : chr  \"C\" \"C\" \"T\" \"A\" ...\n",
      " $ Tumor_Seq_Allele2     : chr  \"G\" \"T\" \"G\" \"G\" ...\n",
      " $ Protein_Change        : chr  \"p.R1028P\" \"p.E178K\" \"p.E666A\" \"p.G245G\" ...\n",
      " $ i_dbNSFP_CADD_phred   : chr  \"17.93\" \"28.5\" \"12.41\" NA ...\n"
     ]
    }
   ],
   "source": [
    "# Let's keep only mutations marked as somatic or unknown and \n",
    "# store the rest in separate data.frame\n",
    "maf.germline.df <- maf.df[! maf.df$Mutation_Status %in% c(\"Somatic\",\"Unknown\"), ]\n",
    "maf.df <- maf.df[ maf.df$Mutation_Status %in% c(\"Somatic\",\"Unknown\"), ]\n",
    "\n",
    "# let's see the structure of data frame\n",
    "str(maf.df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Hugo_Symbol</th><th scope=col>Center</th><th scope=col>NCBI_Build</th><th scope=col>Chromosome</th><th scope=col>Start_position</th><th scope=col>End_position</th><th scope=col>Variant_Classification</th><th scope=col>Variant_Type</th><th scope=col>Tumor_Sample_Barcode</th><th scope=col>Mutation_Status</th><th scope=col>Reference_Allele</th><th scope=col>Tumor_Seq_Allele1</th><th scope=col>Tumor_Seq_Allele2</th><th scope=col>Protein_Change</th><th scope=col>i_dbNSFP_CADD_phred</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>BAI2             </td><td>broad.mit.edu    </td><td>37               </td><td>1                </td><td> 32202221        </td><td> 32202221        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-04-1331-01  </td><td>Unknown          </td><td>C                </td><td>C                </td><td>G                </td><td>p.R1028P         </td><td>17.93            </td></tr>\n",
       "\t<tr><td>LRRC41           </td><td>broad.mit.edu    </td><td>37               </td><td>1                </td><td> 46751997        </td><td> 46751997        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-04-1331-01  </td><td>Somatic          </td><td>C                </td><td>C                </td><td>T                </td><td>p.E178K          </td><td>28.5             </td></tr>\n",
       "\t<tr><td>ERICH3           </td><td>broad.mit.edu    </td><td>37               </td><td>1                </td><td> 75055494        </td><td> 75055494        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-04-1331-01  </td><td>Unknown          </td><td>T                </td><td>T                </td><td>G                </td><td>p.E666A          </td><td>12.41            </td></tr>\n",
       "\t<tr><td>SSX2IP           </td><td>broad.mit.edu    </td><td>37               </td><td>1                </td><td> 85128152        </td><td> 85128152        </td><td>Silent           </td><td>SNP              </td><td>TCGA-04-1331-01  </td><td>Somatic          </td><td>A                </td><td>A                </td><td>G                </td><td>p.G245G          </td><td>NA               </td></tr>\n",
       "\t<tr><td>NTNG1            </td><td>broad.mit.edu    </td><td>37               </td><td>1                </td><td>107866918        </td><td>107866918        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-04-1331-01  </td><td>Somatic          </td><td>G                </td><td>G                </td><td>A                </td><td>p.M87I           </td><td>19.40            </td></tr>\n",
       "\t<tr><td>PGLYRP3          </td><td>broad.mit.edu    </td><td>37               </td><td>1                </td><td>153274927        </td><td>153274927        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-04-1331-01  </td><td>Unknown          </td><td>G                </td><td>G                </td><td>T                </td><td>p.S229Y          </td><td>14.61            </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllllllllllllll}\n",
       " Hugo\\_Symbol & Center & NCBI\\_Build & Chromosome & Start\\_position & End\\_position & Variant\\_Classification & Variant\\_Type & Tumor\\_Sample\\_Barcode & Mutation\\_Status & Reference\\_Allele & Tumor\\_Seq\\_Allele1 & Tumor\\_Seq\\_Allele2 & Protein\\_Change & i\\_dbNSFP\\_CADD\\_phred\\\\\n",
       "\\hline\n",
       "\t BAI2                & broad.mit.edu       & 37                  & 1                   &  32202221           &  32202221           & Missense\\_Mutation & SNP                 & TCGA-04-1331-01     & Unknown             & C                   & C                   & G                   & p.R1028P            & 17.93              \\\\\n",
       "\t LRRC41              & broad.mit.edu       & 37                  & 1                   &  46751997           &  46751997           & Missense\\_Mutation & SNP                 & TCGA-04-1331-01     & Somatic             & C                   & C                   & T                   & p.E178K             & 28.5               \\\\\n",
       "\t ERICH3              & broad.mit.edu       & 37                  & 1                   &  75055494           &  75055494           & Missense\\_Mutation & SNP                 & TCGA-04-1331-01     & Unknown             & T                   & T                   & G                   & p.E666A             & 12.41              \\\\\n",
       "\t SSX2IP            & broad.mit.edu     & 37                & 1                 &  85128152         &  85128152         & Silent            & SNP               & TCGA-04-1331-01   & Somatic           & A                 & A                 & G                 & p.G245G           & NA               \\\\\n",
       "\t NTNG1               & broad.mit.edu       & 37                  & 1                   & 107866918           & 107866918           & Missense\\_Mutation & SNP                 & TCGA-04-1331-01     & Somatic             & G                   & G                   & A                   & p.M87I              & 19.40              \\\\\n",
       "\t PGLYRP3             & broad.mit.edu       & 37                  & 1                   & 153274927           & 153274927           & Missense\\_Mutation & SNP                 & TCGA-04-1331-01     & Unknown             & G                   & G                   & T                   & p.S229Y             & 14.61              \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Hugo_Symbol | Center | NCBI_Build | Chromosome | Start_position | End_position | Variant_Classification | Variant_Type | Tumor_Sample_Barcode | Mutation_Status | Reference_Allele | Tumor_Seq_Allele1 | Tumor_Seq_Allele2 | Protein_Change | i_dbNSFP_CADD_phred | \n",
       "|---|---|---|---|---|---|\n",
       "| BAI2              | broad.mit.edu     | 37                | 1                 |  32202221         |  32202221         | Missense_Mutation | SNP               | TCGA-04-1331-01   | Unknown           | C                 | C                 | G                 | p.R1028P          | 17.93             | \n",
       "| LRRC41            | broad.mit.edu     | 37                | 1                 |  46751997         |  46751997         | Missense_Mutation | SNP               | TCGA-04-1331-01   | Somatic           | C                 | C                 | T                 | p.E178K           | 28.5              | \n",
       "| ERICH3            | broad.mit.edu     | 37                | 1                 |  75055494         |  75055494         | Missense_Mutation | SNP               | TCGA-04-1331-01   | Unknown           | T                 | T                 | G                 | p.E666A           | 12.41             | \n",
       "| SSX2IP            | broad.mit.edu     | 37                | 1                 |  85128152         |  85128152         | Silent            | SNP               | TCGA-04-1331-01   | Somatic           | A                 | A                 | G                 | p.G245G           | NA                | \n",
       "| NTNG1             | broad.mit.edu     | 37                | 1                 | 107866918         | 107866918         | Missense_Mutation | SNP               | TCGA-04-1331-01   | Somatic           | G                 | G                 | A                 | p.M87I            | 19.40             | \n",
       "| PGLYRP3           | broad.mit.edu     | 37                | 1                 | 153274927         | 153274927         | Missense_Mutation | SNP               | TCGA-04-1331-01   | Unknown           | G                 | G                 | T                 | p.S229Y           | 14.61             | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Hugo_Symbol Center        NCBI_Build Chromosome Start_position End_position\n",
       "1 BAI2        broad.mit.edu 37         1           32202221       32202221   \n",
       "2 LRRC41      broad.mit.edu 37         1           46751997       46751997   \n",
       "3 ERICH3      broad.mit.edu 37         1           75055494       75055494   \n",
       "4 SSX2IP      broad.mit.edu 37         1           85128152       85128152   \n",
       "5 NTNG1       broad.mit.edu 37         1          107866918      107866918   \n",
       "6 PGLYRP3     broad.mit.edu 37         1          153274927      153274927   \n",
       "  Variant_Classification Variant_Type Tumor_Sample_Barcode Mutation_Status\n",
       "1 Missense_Mutation      SNP          TCGA-04-1331-01      Unknown        \n",
       "2 Missense_Mutation      SNP          TCGA-04-1331-01      Somatic        \n",
       "3 Missense_Mutation      SNP          TCGA-04-1331-01      Unknown        \n",
       "4 Silent                 SNP          TCGA-04-1331-01      Somatic        \n",
       "5 Missense_Mutation      SNP          TCGA-04-1331-01      Somatic        \n",
       "6 Missense_Mutation      SNP          TCGA-04-1331-01      Unknown        \n",
       "  Reference_Allele Tumor_Seq_Allele1 Tumor_Seq_Allele2 Protein_Change\n",
       "1 C                C                 G                 p.R1028P      \n",
       "2 C                C                 T                 p.E178K       \n",
       "3 T                T                 G                 p.E666A       \n",
       "4 A                A                 G                 p.G245G       \n",
       "5 G                G                 A                 p.M87I        \n",
       "6 G                G                 T                 p.S229Y       \n",
       "  i_dbNSFP_CADD_phred\n",
       "1 17.93              \n",
       "2 28.5               \n",
       "3 12.41              \n",
       "4 NA                 \n",
       "5 19.40              \n",
       "6 14.61              "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "head(maf.df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th></th><th scope=col>Hugo_Symbol</th><th scope=col>Center</th><th scope=col>NCBI_Build</th><th scope=col>Chromosome</th><th scope=col>Start_position</th><th scope=col>End_position</th><th scope=col>Variant_Classification</th><th scope=col>Variant_Type</th><th scope=col>Tumor_Sample_Barcode</th><th scope=col>Mutation_Status</th><th scope=col>Reference_Allele</th><th scope=col>Tumor_Seq_Allele1</th><th scope=col>Tumor_Seq_Allele2</th><th scope=col>Protein_Change</th><th scope=col>i_dbNSFP_CADD_phred</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>20214</th><td>C9orf78          </td><td>broad.mit.edu    </td><td>37               </td><td>9                </td><td>132597484        </td><td>132597484        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-61-2113-01  </td><td>Unknown          </td><td>T                </td><td>T                </td><td>G                </td><td>p.K6T            </td><td>13.48            </td></tr>\n",
       "\t<tr><th scope=row>20215</th><td>CAMSAP1          </td><td>broad.mit.edu    </td><td>37               </td><td>9                </td><td>138714002        </td><td>138714002        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-61-2113-01  </td><td>Unknown          </td><td>G                </td><td>G                </td><td>C                </td><td>p.S835R          </td><td>22.6             </td></tr>\n",
       "\t<tr><th scope=row>20216</th><td>NOTCH1           </td><td>broad.mit.edu    </td><td>37               </td><td>9                </td><td>139391081        </td><td>139391081        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-61-2113-01  </td><td>Unknown          </td><td>G                </td><td>G                </td><td>T                </td><td>p.S2370R         </td><td>12.84            </td></tr>\n",
       "\t<tr><th scope=row>20217</th><td>DPP7             </td><td>broad.mit.edu    </td><td>37               </td><td>9                </td><td>140007856        </td><td>140007856        </td><td>Missense_Mutation</td><td>SNP              </td><td>TCGA-61-2113-01  </td><td>Unknown          </td><td>G                </td><td>G                </td><td>A                </td><td>p.A193V          </td><td>17.85            </td></tr>\n",
       "\t<tr><th scope=row>20218</th><td>FAM47A           </td><td>broad.mit.edu    </td><td>37               </td><td>X                </td><td> 34149721        </td><td> 34149721        </td><td>Silent           </td><td>SNP              </td><td>TCGA-61-2113-01  </td><td>Unknown          </td><td>C                </td><td>C                </td><td>T                </td><td>p.P225P          </td><td>NA               </td></tr>\n",
       "\t<tr><th scope=row>20219</th><td>B3GNT7           </td><td>broad.mit.edu    </td><td>37               </td><td>2                </td><td>232263379        </td><td>232263380        </td><td>In_Frame_Ins     </td><td>INS              </td><td>TCGA-61-2113-01  </td><td>Unknown          </td><td>-                </td><td>-                </td><td>GCG              </td><td>p.317_318insG    </td><td>NA               </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllllllllllllll}\n",
       "  & Hugo\\_Symbol & Center & NCBI\\_Build & Chromosome & Start\\_position & End\\_position & Variant\\_Classification & Variant\\_Type & Tumor\\_Sample\\_Barcode & Mutation\\_Status & Reference\\_Allele & Tumor\\_Seq\\_Allele1 & Tumor\\_Seq\\_Allele2 & Protein\\_Change & i\\_dbNSFP\\_CADD\\_phred\\\\\n",
       "\\hline\n",
       "\t20214 & C9orf78             & broad.mit.edu       & 37                  & 9                   & 132597484           & 132597484           & Missense\\_Mutation & SNP                 & TCGA-61-2113-01     & Unknown             & T                   & T                   & G                   & p.K6T               & 13.48              \\\\\n",
       "\t20215 & CAMSAP1             & broad.mit.edu       & 37                  & 9                   & 138714002           & 138714002           & Missense\\_Mutation & SNP                 & TCGA-61-2113-01     & Unknown             & G                   & G                   & C                   & p.S835R             & 22.6               \\\\\n",
       "\t20216 & NOTCH1              & broad.mit.edu       & 37                  & 9                   & 139391081           & 139391081           & Missense\\_Mutation & SNP                 & TCGA-61-2113-01     & Unknown             & G                   & G                   & T                   & p.S2370R            & 12.84              \\\\\n",
       "\t20217 & DPP7                & broad.mit.edu       & 37                  & 9                   & 140007856           & 140007856           & Missense\\_Mutation & SNP                 & TCGA-61-2113-01     & Unknown             & G                   & G                   & A                   & p.A193V             & 17.85              \\\\\n",
       "\t20218 & FAM47A            & broad.mit.edu     & 37                & X                 &  34149721         &  34149721         & Silent            & SNP               & TCGA-61-2113-01   & Unknown           & C                 & C                 & T                 & p.P225P           & NA               \\\\\n",
       "\t20219 & B3GNT7                & broad.mit.edu         & 37                    & 2                     & 232263379             & 232263380             & In\\_Frame\\_Ins      & INS                   & TCGA-61-2113-01       & Unknown               & -                     & -                     & GCG                   & p.317\\_318insG       & NA                   \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "| <!--/--> | Hugo_Symbol | Center | NCBI_Build | Chromosome | Start_position | End_position | Variant_Classification | Variant_Type | Tumor_Sample_Barcode | Mutation_Status | Reference_Allele | Tumor_Seq_Allele1 | Tumor_Seq_Allele2 | Protein_Change | i_dbNSFP_CADD_phred | \n",
       "|---|---|---|---|---|---|\n",
       "| 20214 | C9orf78           | broad.mit.edu     | 37                | 9                 | 132597484         | 132597484         | Missense_Mutation | SNP               | TCGA-61-2113-01   | Unknown           | T                 | T                 | G                 | p.K6T             | 13.48             | \n",
       "| 20215 | CAMSAP1           | broad.mit.edu     | 37                | 9                 | 138714002         | 138714002         | Missense_Mutation | SNP               | TCGA-61-2113-01   | Unknown           | G                 | G                 | C                 | p.S835R           | 22.6              | \n",
       "| 20216 | NOTCH1            | broad.mit.edu     | 37                | 9                 | 139391081         | 139391081         | Missense_Mutation | SNP               | TCGA-61-2113-01   | Unknown           | G                 | G                 | T                 | p.S2370R          | 12.84             | \n",
       "| 20217 | DPP7              | broad.mit.edu     | 37                | 9                 | 140007856         | 140007856         | Missense_Mutation | SNP               | TCGA-61-2113-01   | Unknown           | G                 | G                 | A                 | p.A193V           | 17.85             | \n",
       "| 20218 | FAM47A            | broad.mit.edu     | 37                | X                 |  34149721         |  34149721         | Silent            | SNP               | TCGA-61-2113-01   | Unknown           | C                 | C                 | T                 | p.P225P           | NA                | \n",
       "| 20219 | B3GNT7            | broad.mit.edu     | 37                | 2                 | 232263379         | 232263380         | In_Frame_Ins      | INS               | TCGA-61-2113-01   | Unknown           | -                 | -                 | GCG               | p.317_318insG     | NA                | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "      Hugo_Symbol Center        NCBI_Build Chromosome Start_position\n",
       "20214 C9orf78     broad.mit.edu 37         9          132597484     \n",
       "20215 CAMSAP1     broad.mit.edu 37         9          138714002     \n",
       "20216 NOTCH1      broad.mit.edu 37         9          139391081     \n",
       "20217 DPP7        broad.mit.edu 37         9          140007856     \n",
       "20218 FAM47A      broad.mit.edu 37         X           34149721     \n",
       "20219 B3GNT7      broad.mit.edu 37         2          232263379     \n",
       "      End_position Variant_Classification Variant_Type Tumor_Sample_Barcode\n",
       "20214 132597484    Missense_Mutation      SNP          TCGA-61-2113-01     \n",
       "20215 138714002    Missense_Mutation      SNP          TCGA-61-2113-01     \n",
       "20216 139391081    Missense_Mutation      SNP          TCGA-61-2113-01     \n",
       "20217 140007856    Missense_Mutation      SNP          TCGA-61-2113-01     \n",
       "20218  34149721    Silent                 SNP          TCGA-61-2113-01     \n",
       "20219 232263380    In_Frame_Ins           INS          TCGA-61-2113-01     \n",
       "      Mutation_Status Reference_Allele Tumor_Seq_Allele1 Tumor_Seq_Allele2\n",
       "20214 Unknown         T                T                 G                \n",
       "20215 Unknown         G                G                 C                \n",
       "20216 Unknown         G                G                 T                \n",
       "20217 Unknown         G                G                 A                \n",
       "20218 Unknown         C                C                 T                \n",
       "20219 Unknown         -                -                 GCG              \n",
       "      Protein_Change i_dbNSFP_CADD_phred\n",
       "20214 p.K6T          13.48              \n",
       "20215 p.S835R        22.6               \n",
       "20216 p.S2370R       12.84              \n",
       "20217 p.A193V        17.85              \n",
       "20218 p.P225P        NA                 \n",
       "20219 p.317_318insG  NA                 "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "tail(maf.df)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "reading maf..\n",
      "Done !\n"
     ]
    }
   ],
   "source": [
    "write.table(maf.df, file = paste0(\"allSamplesMaf.maf\"),\n",
    "            col.names = T, row.names = F, quote = F, sep=\"\\t\")\n",
    "\n",
    "\n",
    "maf <- read.maf(maf = paste0(\"allSamplesMaf.maf\"),\n",
    "                removeSilent = T, useAll = T , verbose = F)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 3.3.2 (2016-10-31)\n",
       "Platform: i386-w64-mingw32/i386 (32-bit)\n",
       "Running under: Windows 7 x64 (build 7601) Service Pack 1\n",
       "\n",
       "locale:\n",
       "[1] LC_COLLATE=Finnish_Finland.1252  LC_CTYPE=Finnish_Finland.1252   \n",
       "[3] LC_MONETARY=Finnish_Finland.1252 LC_NUMERIC=C                    \n",
       "[5] LC_TIME=Finnish_Finland.1252    \n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats     graphics  grDevices utils     datasets  methods  \n",
       "[8] base     \n",
       "\n",
       "other attached packages:\n",
       "[1] maftools_1.0.40     NMF_0.20.6          Biobase_2.34.0     \n",
       "[4] BiocGenerics_0.20.0 cluster_2.0.5       rngtools_1.2.4     \n",
       "[7] pkgmaker_0.22       registry_0.3        readr_1.0.0        \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       " [1] gridExtra_2.2.1            Biostrings_2.42.1         \n",
       " [3] Rsamtools_1.26.1           stringi_1.1.2             \n",
       " [5] evaluate_0.10              memoise_1.0.0             \n",
       " [7] lattice_0.20-34            bitops_1.0-6              \n",
       " [9] RSQLite_1.1-2              survival_2.40-1           \n",
       "[11] trimcluster_0.1-2          cometExactTest_0.1.3      \n",
       "[13] DBI_0.5-1                  data.table_1.10.0         \n",
       "[15] nlme_3.1-130               zlibbioc_1.20.0           \n",
       "[17] GenomicAlignments_1.10.0   ggplot2_2.2.1             \n",
       "[19] rjson_0.2.15               mvtnorm_1.0-5             \n",
       "[21] GlobalOptions_0.0.10       VariantAnnotation_1.20.2  \n",
       "[23] ggrepel_0.6.5              tools_3.3.2               \n",
       "[25] magrittr_1.5               RCurl_1.95-4.8            \n",
       "[27] DEoptimR_1.0-8             IRdisplay_0.4.4           \n",
       "[29] Rcpp_0.12.9                prabclus_2.2-6            \n",
       "[31] IRkernel_0.7.1             assertthat_0.1            \n",
       "[33] R6_2.2.0                   nnet_7.3-12               \n",
       "[35] pbdZMQ_0.2-5               shape_1.4.2               \n",
       "[37] munsell_0.4.3              digest_0.6.12             \n",
       "[39] repr_0.10                  codetools_0.2-15          \n",
       "[41] splines_3.3.2              cowplot_0.7.0             \n",
       "[43] GenomeInfoDb_1.10.2        colorspace_1.3-2          \n",
       "[45] stats4_3.3.2               gridBase_0.4-7            \n",
       "[47] xtable_1.8-2               diptest_0.75-7            \n",
       "[49] iterators_1.0.8            uuid_0.1-2                \n",
       "[51] plyr_1.8.4                 SummarizedExperiment_1.4.0\n",
       "[53] gtable_0.2.0               circlize_0.3.9            \n",
       "[55] wordcloud_2.5              zoo_1.7-14                \n",
       "[57] GetoptLong_0.1.5           modeltools_0.2-21         \n",
       "[59] reshape2_1.4.2             kernlab_0.9-25            \n",
       "[61] ComplexHeatmap_1.12.0      biomaRt_2.30.0            \n",
       "[63] changepoint_2.2.2          IRanges_2.8.1             \n",
       "[65] GenomicRanges_1.26.2       doParallel_1.0.10         \n",
       "[67] flexmix_2.3-13             rtracklayer_1.34.1        \n",
       "[69] AnnotationDbi_1.36.1       tibble_1.2                \n",
       "[71] DPpackage_1.1-6            dendextend_1.4.0          \n",
       "[73] Matrix_1.2-8               scales_0.4.1              \n",
       "[75] BSgenome_1.42.0            RColorBrewer_1.1-2        \n",
       "[77] S4Vectors_0.12.1           fpc_2.1-10                \n",
       "[79] mclust_5.2.2               stringr_1.1.0             \n",
       "[81] slam_0.1-40                dplyr_0.5.0               \n",
       "[83] grid_3.3.2                 class_7.3-14              \n",
       "[85] lazyeval_0.2.0             GenomicFeatures_1.26.2    \n",
       "[87] XML_3.98-1.5               crayon_1.3.2              \n",
       "[89] whisker_0.3-2              MASS_7.3-45               \n",
       "[91] BiocParallel_1.8.1         viridis_0.3.4             \n",
       "[93] jsonlite_1.2               XVector_0.14.0            \n",
       "[95] foreach_1.4.3              robustbase_0.92-7         "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.3.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
